hail/python/hailtop/batch_client/globals.py

```diff
@@ -1,3 +1,5 @@
+ROOT_JOB_GROUP_ID = 0
```

Comment:
Hmm. Why is the first job group id 0 but the first job is id 1?

Reply:
the first job group id ("0") is the root job group. This to me is different than a user-created job group which makes sense to start at 1 just like a user created job. If you want to change this, then we'll need to add new database migrations and it's going to be a huge headache.